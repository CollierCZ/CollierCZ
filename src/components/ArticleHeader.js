import { css } from '@emotion/core'
import React from "react"

class ArticleHeader extends React.Component {
  render() {
    const { children } = this.props;
    return <header
      css={css`
        font-size: 5rem
        margin-bottom: 3.4rem;
        @media only screen and (max-width: 500px) {
            margin-bottom: 2rem;
        }
      `}
    >{children}</header>;
  }
}

export default ArticleHeader;
