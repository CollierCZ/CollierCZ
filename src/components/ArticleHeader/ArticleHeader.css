.article-template .article-header,
.page-template .article-header {
    margin-bottom: 3.4rem;
}

/* ==========================================================================
       Media Queries - Smaller than 500px
   ========================================================================== */

@media only screen and (max-width: 500px) {
    .article-template .article-header,
    .page-template .article-header {
        margin-bottom: 2rem;
    }
}
