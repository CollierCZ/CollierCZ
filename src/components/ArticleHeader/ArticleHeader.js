import React from "react";
import "./ArticleHeader.css";

class ArticleHeader extends React.Component {
  render() {
    const { children } = this.props;
    return <header className="article-header">{children}</header>;
  }
}

export default ArticleHeader;
