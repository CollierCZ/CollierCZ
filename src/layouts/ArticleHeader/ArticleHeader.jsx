import React from "react";
import "./ArticleHeader.css";

class ArticleHeader extends React.Component {
  render() {
    const { children } = this.props;
    return <header className="post-header">{children}</header>;
  }
}

export default ArticleHeader;
