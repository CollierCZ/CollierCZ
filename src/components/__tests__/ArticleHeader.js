import React from "react"
import { shallow } from "enzyme"
import ArticleHeader from "../ArticleHeader"

describe("ArticleFormating", () => {
  it("renders correctly", () => {
    const component = shallow (<ArticleHeader children={{test: "value1"}} />);
    expect(component).toMatchSnapshot()
  })
})