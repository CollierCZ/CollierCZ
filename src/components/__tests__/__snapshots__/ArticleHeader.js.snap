// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`ArticleFormating renders correctly 1`] = `
<header
  css={
    Object {
      "map": undefined,
      "name": "onsyne",
      "next": undefined,
      "styles": "
        margin-bottom: 3.4rem;
        @media only screen and (max-width: 500px) {
            margin-bottom: 2rem;
        }
      ",
    }
  }
>
  <Component />
</header>
`;
